test/crowdsale/PausableCrowdsale.test.js

```diff
@@ -0,0 +1,153 @@
+const shouldFail = require('../helpers/shouldFail');
+const expectEvent = require('../helpers/expectEvent');
+
+const PausableCrowdsale = artifacts.require('PausableCrowdsaleImpl');
+const SimpleToken = artifacts.require('SimpleToken');
+
+require('../helpers/setup');
+
+contract('PausableCrowdsale', function ([owner, wallet, stranger]) {
```

Comment:
```suggestion
contract('PausableCrowdsale', function ([_, owner, wallet, stranger]) {
```

See #1137

Reply:
Also, there is no `owner`, only a `pauser`. And we're calling that third account `anyone`.